BCH-Wunder-Project-team-4/Wunder.io
# Cookie bar

## Link to ticket:

[Cookie ticket](https://trello.com/c/HanXBSyK/55-cookie-bar)

## Changes proposed in this PR:

- Cookie bar! Selection of some options
- Appears on any page if cookie consent has not been given yet. Once something is selected it will not show up until going to cookie settings or clearing cookies manually in the browser.
- Installed js-cookies

## How to test:

1. git fetch origin cookieBar
2. git checkout cookieBar
3. npm install
4. cookie bar should appear now on any page
5. after selection it shouldn't appear again
6. Cookie Settings in footer should reopen it on reload
7. You can check the true/false state of the cookie selections in inspect tool -> application -> cookies


<img width="1094" alt="Screenshot 2023-12-15 at 15 57 59" src="https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/121946942/079a7670-80f6-4de8-bb57-a9738710b72a">

<img width="553" alt="Screenshot 2023-12-15 at 16 06 32" src="https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/121946942/f60e4edc-5572-478a-8196-01cc852ee522">
